src/maxdiffusion/max_utils.py

```diff
@@ -433,3 +433,53 @@ def get_memory_allocations():
     m_stats = device.memory_stats()
     max_logging.log(f'device : {device.process_index},'
                     f'bytes in use: {m_stats["bytes_in_use"] / gb} / {m_stats["bytes_limit"] / gb} GB')
+
+def calculate_training_tflops(pipeline, unet_params, config):
```

Comment:
how about having an argument in the function - `training`/`inference`? and also have an argument about factors like 3 and 3.2?

Reply:
will do it in another commit.